src/stirling/source_connectors/socket_tracer/protocols/common/data_stream_buffer_test.cc

```diff
@@ -152,6 +152,15 @@ TEST_P(DataStreamBufferTest, Timestamp) {
   EXPECT_EQ(stream_buffer.Head(), "123456789");
   EXPECT_OK_AND_EQ(stream_buffer.GetTimestamp(8),
                    5);  // timestamp is adjusted to previous timestamp + 1
```

Comment:
Is the test code we added last time relevant still? Do we need 3 non monotonic `Add` calls or are the ones you added in this PR enough?